### ISSUE TITLE
Tooltip should be able to be disassociated from it's trigger

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#5250]
### Description

In order to build more accessible experiences, Tooltip should be able to be disassociated with it's trigger element. An example of where this would be useful is an icon button:

```tsx
<Tooltip content="Settings" >
<Button aria-label="Settings"><SettingsIcon aria-hidden /></Button>
</Tooltip>
```

On a screenreader this would read "Settings" twice.

### Expected behavior

.

### Alternatives and examples

.

### Context

.

### Are you a JPMorgan Chase & Co. employee?

- [ ] I am an employee of JPMorgan Chase & Co.
